Source/WebInspectorUI/UserInterface/Views/SettingsTabContentView.js

```diff
@@ -279,6 +279,12 @@ WI.SettingsTabContentView = class SettingsTabContentView extends WI.TabContentVi
 
         let elementsSettingsView = new WI.SettingsView("elements", WI.UIString("Elements"));
 
+        if (InspectorBackend.hasCommand("DOM.setInspectModeEnabled", "showRulers")) {
+            elementsSettingsView.addSetting(WI.UIString("DOM Tree:"), WI.settings.domTreeDimsUnrenderedNodes, WI.UIString("Dim unrendered nodes"));
```

Comment:
I think maybe the text could be a bit more verbose to help developers understand.  Maybe something like

> De-emphasize nodes that are not rendered.

or 

> Differentiate nodes that will never paint.

(and I'd probably rename the `WI.Setting` to match)